master/buildbot/test/unit/test_schedulers_base.py

```diff
@@ -158,9 +158,10 @@ def gotChange(got_change, got_important):
         d = sched.startConsumingChanges(**kwargs)
 
         def test(_):
-            # check that it registered a callback
-            self.assertEqual(len(self.mq.qrefs), 1)
-            qref = self.mq.qrefs[0]
+            # check that it registered callbacks
+            self.assertEqual(len(self.mq.qrefs), 2)
+
+            qref = self.mq.qrefs[1]
             self.assertEqual(qref.filter, ('changes', None, 'new'))
```

Comment:
There is a need to test that the scheduler correctly react to the disable events